enclave-runtime/Cargo.lock

```diff
@@ -3464,7 +3479,17 @@ version = "0.3.0"
 source = "git+https://github.com/mesalock-linux/serde-big-array-sgx#94122c5167aee38b39b09a620a60db2c28cf7428"
 dependencies = [
  "serde 1.0.118 (git+https://github.com/mesalock-linux/serde-sgx)",
- "serde_derive 1.0.118",
+ "serde_derive 1.0.118 (git+https://github.com/mesalock-linux/serde-sgx)",
+]
+
+[[package]]
+name = "serde_derive"
```

Comment:
What I did, was add the `tag` to the `serde_derive` import, because all our other imports have that tag as well. My initial issue, was that I could not write a proper test with de-serialization of `RpcResponse` in the enclave, because some `serde` problems with `sgx`/`std`. Making our imports consistent (with the tag), was something I tried, but it didn't help. I left it in though, because of consistency.

Reply:
Change is reverted, and that additional `serde_derive` disappeared again 👍